terraform/modules/cluster/slurm/outputs.tf

```diff
@@ -0,0 +1,17 @@
+output "project_id" { value = var.project_id }
+output "resource_prefix" { value = var.resource_prefix }
+output "filestore_new" { value = var.filestore_new }
+output "gcsfuse_existing" { value = var.gcsfuse_existing }
```

Comment:
do we need all these outputs ?

Reply:
I was just mimicking the functionality of resources in terraform. the first line in the [attributes section](https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/compute_instance_template#attributes-reference) always says:
> In addition to the arguments listed above, the following computed attributes are exported:

so I made all of the inputs outputs. do people not do that with modules?